packages/app/src/domain/layout/layout.tsx

```diff
@@ -36,6 +37,13 @@ export function getLayoutWithMetadata(metadata: LayoutProps) {
   };
 }
 
+export function getDefaultLayout() {
+  return function (page: React.ReactNode, pageProps: any) {
```

Comment:
Can we type it something like this? So we'll validate it won't break on a missing `lastGenerated` prop
```suggestion
  return function<T extends { lastGenerated: string }>(page: React.ReactNode, pageProps: T) {
```